tests/phpunit/Mutation/FileMutationGeneratorTest.php

```diff
@@ -150,18 +147,6 @@ public function test_it_attempts_to_generate_mutations_for_the_file_if_covered_o
         $this->traverserFactoryMock
             ->expects($this->once())
             ->method('create')
-            ->with($this->callback(function (array $passedExtraNodeVisitors) use ($extraVisitors) {
```

Comment:
I'm not really sure what was going on here, but i think with my changes this is safe to delete now

Reply:
IIRC it was to check that the default priority was used. Since we don't care about that anymore, I would just stick with the standard `with()` to make sure the right parameters are passed